scripts/alcf-builds/sycl.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+###############################################################################
+# Copyright (c) 2017-24, Lawrence Livermore National Security, LLC
+# and RAJA project contributors. See the RAJAPerf/LICENSE file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+###############################################################################
+
+
+BUILD_SUFFIX=sycl
+: ${BUILD_TYPE:=RelWithDebInfo}
+RAJA_HOSTCONFIG=../tpl/RAJA/host-configs/alcf-builds/sycl.cmake
+
+rm -rf build_${BUILD_SUFFIX}_${USER} >/dev/null
+mkdir build_${BUILD_SUFFIX}_${USER} && cd build_${BUILD_SUFFIX}_${USER}
+
+cmake \
+  -DCMAKE_BUILD_TYPE=${BUILD_TYPE} \
+  -C ${RAJA_HOSTCONFIG} \
+  -DENABLE_OPENMP=Off \
+  -DENABLE_CUDA=Off \
+  -DRAJA_PERFSUITE_GPU_BLOCKSIZES=0,64,128,256,512,1024 \
```

Comment:
Do we handle 0 properly everywhere?

Reply:
We agreed to remove those. So I changed the script.